### DISCUSSION TITLE
Swoole Promise Adapter

### DISCUSSION THREAD

**Discussion by mcharytoniuk** [webonyx/graphql-php#1477] [Show and tell]
I recently finished async promise adapter that supports Swoole. It's a part of a bigger project, but can be used standalone:
https://resonance.distantmagic.com/docs/features/graphql/standalone-promise-adapter.html 

It also supports [Swoole Futures](https://resonance.distantmagic.com/docs/features/swoole-futures/standalone-usage.html) for that extra parallelism. :)

Would it be ok if I made a PR and referece it on this page? https://webonyx.github.io/graphql-php/data-fetching/#async-php

**Comment by spawnia** [comment]
Nice, I would appreciate an update to the docs very much.
